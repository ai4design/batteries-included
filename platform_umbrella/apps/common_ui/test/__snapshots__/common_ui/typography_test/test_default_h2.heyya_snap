<h2 class="text-xl sm:text-3xl font-semibold my-3 text-black">
  Test H2 Header
</h2>