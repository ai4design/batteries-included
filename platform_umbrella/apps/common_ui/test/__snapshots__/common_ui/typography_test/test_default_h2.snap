"<h2 class=\"text-2xl sm:text-3xl font-bold leading-10 text-primary-500\">\n  Test H2 Header\n</h2>"